views/replication-controller.html

```diff
@@ -8,7 +8,7 @@
     <dt>Created</dt>
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
     <dt>Replicas</dt>
-    <dd>{{resource.spec.replicas}}</dd>
+    <dd>{{resource.spec.replicas || 0}}</dd>
```

Comment:
Careful... the default changed in `v1` :-/
In v1beta3, missing means "0". In v1, missing means "1". Add a TODO to change when we migrate to v1